nilearn/image/tests/test_image.py

```diff
@@ -241,7 +243,13 @@ def test_mean_img():
         with testing.write_tmp_imgs(*imgs) as imgs:
             mean_img = image.mean_img(imgs)
             assert_array_equal(mean_img.get_affine(), affine)
-            assert_array_equal(mean_img.get_data(), truth)
+            if X64:
+                assert_array_equal(mean_img.get_data(), truth)
+            else:
```

Comment:
I'll merge this and add a comment that we don't really understand why we have to do this for 32bit but we don't think it is a priority to investigate.